features/lessons/src/main/java/com/cryart/sabbathschool/lessons/ui/readings/SSReadingViewModel.kt

```diff
@@ -169,6 +168,16 @@ class SSReadingViewModel(
             })
     }
 
+    private fun parseDate(date: String): DateTime? {
+        return try {
+            DateTimeFormat.forPattern(SSConstants.SS_DATE_FORMAT)
+                .parseLocalDate(date).toDateTimeAtStartOfDay()
+        } catch (ex: Exception) {
+            Timber.e(ex)
```

Comment:
Please include `date` in Timber message so we can find it in content.

Reply:
The date will be in the exception message.
It looks like this:
`Cannot parse "31/04/2021": Value 31 for dayOfMonth must be in the range [1,30]`